mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,139 @@
+
+
+## Workshops
+
+(Currently under construction:)
+For workshops or courses that do not have a Course Code, you need to provide us with the course or workshop name and a list of all vsc-ids. A group will be made, based on the name of the workshop, with all vsc-ids as member. Teachers/moderators will be able to add/remove vsc accounts from this course group. But students will have to follow the [procedure to request a vsc account]{ https://docs.hpc.ugent.be/account/} themselves. There will be no automation.
+
+
+## Group moderators
+
+The course group is used a.o. to manage storage permissions. Moderators are the owners of a group and have raised privileges: e.g. they have read/write access in specific folders, can assign subgroups, etc.
+Typically, the teachers map to the role of moderators of a course group, and students are regular members.
+Provide us with a list of all the vsc accounts for the teachers of your course. 
+
+
+## Start and end date (and time) for your course/workshop
+
+The end date is used to automatically perform a cleanup when your course/workshop has finished:
+- Course group and subgroups will be deactivated
+- Reservations will be terminated
+- Residual data in the course directories will be deleted
+- Custom Interactive Applications will be disabled
+
+The start date (and time) is used as a target for the HPC-UGent team to set up your course requirements. But note that this target is best-effort, depending on the load of the support team and the complexity of your requirements. 
+Requests should be made well in advance, at least several weeks before the actual start of your course. The sooner you make the request, the better.
+
+
+## Dedicated course storage
+
+For every course, a dedicated course directory will be created on the DATA filesystem under: /data/gent/courses/<year>/<coursecode>_<year>
+e.g. /data/gent/courses/2023/e071400_2023
+
+This directory will be accessible by all members of your course group. (Hence, it is no longer necessary to set up or invite course members to your virtual organization.)
+Every course directory will always contain the folders:
+- /input
+  - ideally suited to distribute initial common datasets or input data
+  - moderators have read/write access
+  -  group members (students) only have read access
+- /members
+  - this directory contains a personal folder for every student in your course:
+  /members/vsc<4abcd>
+  - only this specific vsc-id will have read/write access to this folder
+  - moderators have read access to this folder
+
+Optionally, we can also create these folders:
+- /shared
+  - this is a folder for sharing files between any and all group members
+  - all group members and moderators have read/write access
+  - beware that group members will be able to alter/delete each others files in this folder
```

Comment:
well, the sticky bit is set so this is not completely true, but i'm not sure how to nuance this without making it complicated

Reply:
i'm not sure there will be (many) use cases for this directory since we have support for subgroups, but time will tell i guess